### PR TITLE
Json schema validation added.

### PR DESCRIPTION
**Pull request by MelissaRoman06**

### CODE REVIEW

**Review by alszla — APPROVED** [review]
Everything looks fine, there was only minor changes in the only java file of this commit.

The rest of the files are feature or schema files. Feature files were properly indented and json schemas were formatted properly as well.

File: src/test/resources/schemas/LeadErrorFieldsSchema.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        }
+      },
+      "required": [
+        "message",
+        "errorCode",
+        "fields"
+      ]
+    }
+  ]
+}
```
Comment (titor-oopart) [inline src/test/resources/schemas/LeadErrorFieldsSchema.json]: add empty line

File: src/test/resources/schemas/LeadErrorSchema.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          "type": "string"
+        }
+      },
+      "required": [
+        "errorCode",
+        "message"
+      ]
+    }
+  ]
+}
```
Comment (titor-oopart) [inline src/test/resources/schemas/LeadErrorSchema.json]: add empty line

File: src/test/resources/schemas/LeadFindSchema.json [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  "CompanyDunsNumber": null,
+  "DandbCompanyId": null,
+  "EmailBouncedReason": null,
+  "EmailBouncedDate": null,
+  "SICCode__c": null,
+  "ProductInterest__c": null,
+  "Primary__c": null,
+  "CurrentGenerators__c": null,
+  "NumberofLocations__c": null
+}
```
Comment (titor-oopart) [inline src/test/resources/schemas/LeadFindSchema.json]: add empty line

File: src/test/resources/schemas/LeadSummarySchema.json [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+          ]
+        }
+      ]
+    }
+  },
+  "required": [
+    "objectDescribe",
+    "recentItems"
+  ]
+}
```
Comment (titor-oopart) [inline src/test/resources/schemas/LeadSummarySchema.json]: add empty line

**Review by Cxristian — APPROVED** [review]

**Review by titor-oopart — APPROVED** [review]

**Pull request merged**